src/services/tech-radar/client.ts

```diff
@@ -0,0 +1,16 @@
+interface TechRadarClient {
```

Comment:
We're missing the export of this interface.

```suggestion
export interface TechRadarClient {
```